roles/common/defaults/main.yml

```diff
@@ -5,7 +5,7 @@ duf_version: "0.8.1"
 dust_version: "0.9.0"
 fd_find_version: "9.0.0"
 configured_dependencies: []
-common_packages:
+common_ubuntu_packages:
```

Comment:
There's a lot of overlap between the Ubuntu and Rocky package lists - would it make sense to make a `shared_packages` list and then only have what's not in common in the Ubuntu and Rocky lists?